spec/features/posts/posts_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #show' do
+    it 'returns http success' do
+      get users_path(1)
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #users' do
+    it 'returns http success' do
+      get users_path(1)
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] You should have the `posts_index_spec` file tests:

  - that asserts you can see the user's profile picture.
  - that asserts you can see the user's username.
  - that asserts you can see the number of posts the user has written.
  - that asserts you can see a post's title.
  -  that asserts you can see some of the post's body.
  -  that asserts you can see the first comments on a post.
  -  that asserts you can see how many comments a post has.
  -  that asserts you can see how many likes a post has.
  - that asserts you can see a section for pagination if there are more posts than fit on the view.
  -  that redirects to the post's show page when you click on a post.

- [ ] Please kindly add all these tests to your file, those tests are for the `User` post index page 🙃